ext/mysql2/extconf.rb

```diff
@@ -113,7 +113,7 @@ def add_ssl_defines(header)
   warn "-----\nUsing --with-mysql-dir=#{File.dirname inc}\n-----"
   rpath_dir = lib
   have_library('mysqlclient')
-elsif (mc = (with_config('mysql-config') || Dir[GLOB].first))
+elsif (mc = with_config('mysql-config') || Dir[GLOB].first)
```

Comment:
I ran `rubocop -a` and it did this, figured I might as well leave it.

Reply:
Very old line of code!